src/ResourceManager/Resources/Commands.Resources/help/Get-AzureRmDenyAssignment.md

```diff
@@ -0,0 +1,370 @@
+```
+
+### ResourceGroupWithSPNParameterSet
+```
+Get-AzureRmDenyAssignment -ServicePrincipalName <String> -ResourceGroupName <String> [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+### ResourceWithSPNParameterSet
+```
+Get-AzureRmDenyAssignment -ServicePrincipalName <String> -ResourceGroupName <String> -ResourceName <String> -ResourceType <String> [-ParentResource <String>]
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+### ScopeWithSPNParameterSet
+```
+Get-AzureRmDenyAssignment -ServicePrincipalName <String> -Scope <String> [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+### SPNParameterSet
+```
+Get-AzureRmDenyAssignment -ServicePrincipalName <String> [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+### ResourceGroupParameterSet
+```
+Get-AzureRmDenyAssignment -ResourceGroupName <String> [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+### ResourceParameterSet
+```
+Get-AzureRmDenyAssignment -ResourceGroupName <String> -ResourceName <String> -ResourceType <String> [-ParentResource <String>]
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+### ScopeParameterSet
+```
+Get-AzureRmDenyAssignment -Scope <String> [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Use the Get-AzureRmDenyAssignment command to list all deny assignments that are effective on a scope.
+Without any parameters, this command returns all the deny assignments made under the subscription.
+This list can  be filtered using filtering parameters for principal, deny assignment name and scope.
+To specify a user, use SignInName or Azure AD ObjectId parameters.
+To specify a security group, use Azure AD ObjectId parameter.
+And to specify an Azure AD application, use ServicePrincipalName or ObjectId parameters.
+The scope at which access is being denied may be specified.
+It defaults to the selected subscription. 
+The scope of the deny assignment can be specified using one of the following parameter combinations
+        a.
+Scope - This is the fully qualified scope starting with /subscriptions/\<subscriptionId\>.
+This will filter deny assignments that are at, above and below that particular scope.
+        b.
+ResourceGroupName - Name of any resource group under the subscription.
+This will filter deny assignments at, above and below the specified resource group
+        c.
+ResourceName, ResourceType, ResourceGroupName and (optionally) ParentResource - Identifies a particular resource under the subscription and will filter deny assignments effective at that resource scope.
+To determine what access is denied for a particular user in the subscription, use the ExpandPrincipalGroups switch.
+This will list all deny assignments assigned to the user, and to the groups that the user is member of.
+
+## EXAMPLES
```

Comment:
@kirankulkarni747 for the examples you've given below, would you please provide the corresponding output from their execution? This will give users a better idea of what properties are found on the returned object

Reply:
Ok added sample outputs.